ui/src/app/applications/components/application-sync-options.tsx

```diff
@@ -31,22 +31,65 @@ function booleanOption(name: string, label: string, defaultVal: boolean, props:
     );
 }
 
-const optionStyle: React.CSSProperties = {marginTop: '0.5em'};
+enum ManualSyncFlags {
+    Prune = 'Prune',
+    DryRun = 'Dry Run',
+    ApplyOnly = 'Apply Only',
+    Force = 'Force'
+}
+
+enum SyncOptions {
+    Validate = 'Skip Schema Validation',
```

Comment:
I think `Validate` is handled incorrectly. Unlike other options validate default value is true, so when it is checked we need to add `Validate=false` (currently UI adds `Validate=true`)